apps/backend/src/lambdas/reconcile.ts

```diff
@@ -32,33 +39,66 @@ export const handler = async (event: ReconciliationConfigInput) => {
           LocationMethod.Bank
         );
 
-  const cashDepositService = app.get(CashDepositService);
-  const dates: Date[] = eachDayOfInterval({
-    start: parse(event.period.from, 'yyyy-MM-dd', new Date()),
-    end: parse(event.period.to, 'yyyy-MM-dd', new Date()),
-  });
+  const masterLocations = await locationService.getLocationsByID(
```

Comment:
I think we can use one call to the location service to get this result. If we remove the "LocationMethod" enum (which is confusing anyways, lol, my bad!) and get all locations regardless of the method, then we can call the reduce function on that first call, rather than querying twice for locations?